.github/actions/setup-environment/action.yml

```diff
@@ -36,21 +36,25 @@ runs:
       run: |
         make build-tests
       shell: bash
+      working-directory: "./eden_repo"
```

Comment:
Do we need to provide `working-directory` for every step in every custom action or will it be overridden by high level `working-directory` parameter i.e. in `test.yml` workflow?

Reply:
It's a good suggestion. Let me try. As per github docs, The `working-directory` attribute will be set only for the `run` section so it should not conflict with the `path`  attribute of the `uses` checkout action

Reply:
I am trying to fix it at the test.yml workflow level :). Let's see if it works. If not then defaults at the action level

Reply:
The `defaults` sections is not supported in actions. Need to set the `shell: bash` and `working-directory: "./eden_repo"` for all the `run` commands explicitly.

Reply:
Well, at least we tried :D